libs/ui/src/admin/TimelineBoard/TimelineBoard.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react'
+
+import { Box, HStack, Link, Text } from '@chakra-ui/react'
+
+import { TimelineTweetBase } from '../TimelineTweet'
+import { TimelineBoardProps } from './types'
+
+export const TimelineBoard: FC<TimelineBoardProps> = ({ timelines }) => {
+  const onEdit = () => {
+    console.log('edit')
+  }
+
+  const onSave = () => {
+    console.log('save')
+  }
+
+  console.log(timelines)
+  return (
+    <HStack
+      align="start"
+      bg={'white'}
+      rounded="lg"
+      p={4}
+      gap={4}
+      overflowY="auto"
+      shouldWrapChildren={true}
+    >
+      {timelines?.map(timeline => (
+        // eslint-disable-next-line react/jsx-key
+        <Box
+          w="500px"
```

Comment:
React requires unique key prop for the elements in a list!

```suggestion
        <Box
          key={timeline.id}
          w="500px"
```